pkg/bench/benchprof/benchprof.go

```diff
@@ -23,8 +23,35 @@ import (
 type StopFn func(testing.TB)
 
 // Stop stops profiling.
-func (f StopFn) Stop(b testing.TB) {
-	f(b)
+func (f StopFn) Stop(tb testing.TB) {
+	f(tb)
```

Comment:
This is fine, but you can also `if f != nil { f(tbg) }` and then you can use `nil` as a noop stop function, which you could use in 2-3 places here.